testnets/manifestdevnet/assetlist.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "manifestdevnet",
+  "assets": [
+    {
+      "description": "Manifest devnet native token",
+      "denom_units": [
+        {
+          "denom": "umfx",
+          "exponent": 0
+        },
+        {
+          "denom": "mfx",
+          "exponent": 6
+        }
+      ],
+      "base": "umfx",
+      "name": "Manifest Devnet Token",
+      "display": "mfx",
+      "symbol": "MFX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.svg"
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Proof of Authority token for the Manifest devnet",
+      "denom_units": [
+        {
+          "denom": "upoa",
+          "exponent": 0
+        },
+        {
+          "denom": "poa",
+          "exponent": 6
+        }
+      ],
+      "base": "upoa",
+      "name": "Manifest Devnet Authority Token",
+      "display": "poa",
+      "symbol": "POA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/manifest/images/manifest.png",
```

Comment:
This directory does not exist, but more importantly, we should reference the manifest images that already exist, and not upload duplicate images.